src/cryptonote_basic/account.cpp

```diff
@@ -50,6 +51,24 @@ DISABLE_VS_WARNINGS(4244 4345)
 
   namespace cryptonote
 {
+ account_keys& account_keys::operator=(const account_keys &keys) {
```

Comment:
C++11 has `account_keys& operator=(account_keys const&) = default;` that can be placed in the declaration, which is a bit easier.

Reply:
done